src/huntsman/pocs/dome/musca.py

```diff
@@ -269,7 +269,7 @@ def _get_shutter_status_dict(self):
         self._write_musca(Protocol.GET_STATUS)
         shutter_status_dict = {}
         num_lines = len(Protocol.VALID_DEVICE)
-        for i in range(num_lines + 1):
+        for i in range(num_lines + 1):  # Add one for the beginning 'Status' key
```

Comment:
I probably should have mentioned this in the last PR, but it occurs to me now that we should probably not put the `Status` key in the `shutter_status_dict`.

So after getting `k, v` below, just a 

```
if k == "Status":
    continue
```

or some such.